docs/core/preview3/tools/csproj.md

```diff
@@ -19,19 +19,80 @@ This document outlines the changes that were added to the csproj files as part o
 [MSBuild](https://github.com/Microsoft/MSBuild). For more information about general project file syntax and reference, 
 see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/msbuild-project-file-schema-reference) documentation.  
 
+## Implicit package references
+The metapackage included is tied to the target framework. That means that for `netcoreapp1.0` a proper version of the `Microsoft.NETCore.App` metapackage is referenced. Same goes for `netcoreapp1.1` target framework as well as `netstandard1.x` target frameworks. 
+
+As far as the rest of the behavior is concerned, the tools will work as expected and most of the gestures will remain the same (for example, `dotnet restore`). 
+
+### Migrating a project
+If you have an existing reference to the metapackage in your project.json, the migration (both in `dotnet migrate` as well as Visual Studio 2017) will include that reference in your new csproj project. This will cause the tools to issue the following warning when you try to build your project:
+
+> A PackageReference for [metapackage ID] was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see https://aka.ms/sdkimplicitrefs.
+
+This warning simply means that you probably want to remove that package reference from your project file. If you do keep it, the specified version of the metapackage will be used. 
+
+### Recommendations
+With this new feature, you might be wondering when you should specify a version of the metapackage in the project and when not. Here is the overall guidance:
+
+* For new projects, you should use the template and not add an explicit reference to any metapackage. 
+* For existing projects, you should remove the reference. 
+* If you need a specific version of the runtime, you should use the `<RuntimeFrameworkVersion>` property in your project (for example, `1.0.4`) instead of referencing the metapackage.
+    * This might happen if you are using [self-contained deployments](https://docs.microsoft.com/en-us/dotnet/articles/core/preview3/deploying/#self-contained-deployments-scd) and you need a specific patch version of 1.0.0 LTS runtime, for example.
+* If you need a specific version of the `NetStandard.Library` metapackage, you can use the `<NetStandardImplicitPackageVersion>` property and set the version you need. 
+
+## Default compilation includes in .NET Core projects
+As part of the RC3 release of Visual Studio 2017, a new version of the .NET Core SDK is included. With that, we've  moved the default includes and excludes for compile items and embedded resources to the SDK properties files. This means that you don't need to specify these items in your project file moving forward. 
```

Comment:
Why do we need to call out the RC3 release and the new version of the SDK. isn't this whole topic about that? 
Maybe "With the move to the csproj format, we've moved the default....

moving forward -> anymore